EIPS/eip-5773.md

```diff
@@ -0,0 +1,362 @@
+---
+eip: 5773
+title: Multi-Resource Token
```

Comment:
Probably a bit too vague. Try to expand a tiny bit about what resources are, or what you can do with these tokens while staying under the character limit.